scss/mixins/_buttons.scss

```diff
@@ -10,16 +10,16 @@
     border-color: $focus-border;
   }
 
-  &:focus,
-  &.focus {
+  #{if(&, "&", "*")}:focus,
+  #{if(&, "&", "*")}.focus {
     color: $color;
```

Comment:
This is a good example, this mixin is _always_ meant to be inside `.btn`, and if used outside would have unwanted side effects.

Reply:
u saved my time :)